package.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
This is where we specify all of our ``node`` dependencies for things we run at the command line.